src/plugins/image_embeddable/public/image_editor/image_editor_flyout.tsx

```diff
@@ -0,0 +1,461 @@
+                    height: 100%;
+                  }
+                `}
+                iconType="image"
+                color="subdued"
+                title={
+                  <p>
+                    <FormattedMessage
+                      id="imageEmbeddable.imageEditor.byURLNoImageTitle"
+                      defaultMessage="No Image"
+                    />
+                  </p>
+                }
+                titleSize={'s'}
+                body={
+                  <p>
+                    <FormattedMessage
+                      id="imageEmbeddable.imageEditor.byURLNoImageMessage"
+                      defaultMessage="Insert a valid URL to the image in the text field below."
+                    />
+                  </p>
+                }
+              />
+            ) : (
+              <ImageViewer
+                css={CONTAINER_SIZING_CSS}
+                imageConfig={draftImageConfig}
+                onError={() => {
+                  setSrcUrlError(failedToLoadImageFromURL(srcUrl));
+                }}
+                containerCSS={css`
+                  border: ${euiTheme.border.thin};
+                  background-color: ${euiTheme.colors.lightestShade};
+                `}
+              />
+            )}
+
+            <EuiSpacer />
+            <EuiFormRow
+              label={
+                <FormattedMessage
+                  id="imageEmbeddable.imageEditor.imageURLInputLabel"
+                  defaultMessage="Insert a URL to the image"
+                />
+              }
+              helpText={
+                <FormattedMessage
+                  id="imageEmbeddable.imageEditor.imageURLHelpText"
+                  defaultMessage="Example: https://elastic.co/my-image.png"
+                />
+              }
+              fullWidth={true}
+              isInvalid={!!srcUrlError}
+              error={srcUrlError}
+            >
+              <EuiTextArea
+                data-test-subj={'imageEmbeddableEditorUrlInput'}
+                fullWidth
+                compressed={true}
+                placeholder={i18n.translate('imageEmbeddable.imageEditor.imageURLPlaceholderText', {
+                  defaultMessage: 'Example: https://elastic.co/my-image.png',
+                })}
+                value={srcUrl}
+                onChange={(e) => {
+                  const url = e.target.value;
+
+                  const { isValid, error } = props.validateUrl(url);
+                  if (!isValid) {
+                    setSrcUrlError(error!);
+                  } else {
+                    setSrcUrlError(null);
+                  }
+
+                  setSrcUrl(e.target.value);
+                }}
+              />
+            </EuiFormRow>
+          </>
+        )}
+        <EuiSpacer />
+        <EuiFormRow
+          label={
+            <FormattedMessage
+              id="imageEmbeddable.imageEditor.imageSizingLabel"
+              defaultMessage="Sizing"
```

Comment:
I decided to change to `Fill mode` because realized that this is how we have it canvas:

![Screenshot 2022-12-08 at 14 57 27](https://user-images.githubusercontent.com/7784120/206464464-28fc0059-3433-4269-8694-d2e279b33739.png)

Reply:
Yes, good to align with Canvas.

I'd change the tooltip, though:

> Stretch mode might not work with vector images